controlplane/kubeadm/api/v1beta1/kubeadm_control_plane_webhook.go

```diff
@@ -431,7 +431,9 @@ func paths(path []string, diff map[string]interface{}) [][]string {
 	for key, m := range diff {
 		nested, ok := m.(map[string]interface{})
 		if !ok {
-			allPaths = append(allPaths, append(path, key))
+			tmp := make([]string, len(path))
+			copy(tmp, path)
+			allPaths = append(allPaths, append(tmp, key))
```

Comment:
nit: @yongxiu Please add a short comment that ~ we have to use a copy of path, because otherwise the slice we append to allPaths would be overwritten in another iteration